src/ResourceManager/Network/Commands.Network/VirtualNetwork/VirtualNetworkListUsageMethod.cs

```diff
@@ -0,0 +1,65 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using AutoMapper;
+using CNM = Microsoft.Azure.Commands.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network.Automation
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmVirtualNetworkUsageList"), OutputType(typeof(List<PSVirtualNetworkUsage>))]
+    public partial class GetAzureRmVirtualNetworkUsageList : NetworkBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource group name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Virtual network name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
+            var vVirtualNetworkList = this.NetworkClient.NetworkManagementClient.VirtualNetworks.ListUsage(ResourceGroupName, Name);
+            var vnetUsageList = new List<PSVirtualNetworkUsage>();
+            foreach (var vVirtualNetwork in vVirtualNetworkList)
+            {
+                var vVirtualNetworkModel = Mapper.Map<CNM.PSVirtualNetworkUsage>(vVirtualNetwork);
+                vnetUsageList.Add(vVirtualNetworkModel);
+            }
+            WriteObject(vnetUsageList);
```

Comment:
@EvgenyAgafonchikov this should be `WriteObject(vnetUsageList, true)` (which sets the `enumerateCollection` parameter to true) so that the list you have constructed writes each object to the output stream rather than writes the array as a whole object to the stream